client/packages/invoices/src/OutboundShipment/ListView/ListView.tsx

```diff
@@ -58,75 +41,35 @@ const OutboundShipmentListViewComponent: FC = () => {
       { key: 'invoiceNumber', condition: 'equalTo', isNumber: true },
     ],
   });
-  const navigate = useNavigate();
-  const modalController = useToggle();
-  const pagination = { page, first, offset };
   const queryParams = { ...filter, sortBy, first, offset };
-  const simplifiedTabletView = useSimplifiedTabletUI();
-
-  const { data, isError, isLoading } = useOutbound.document.list(queryParams);
-  useDisableOutboundRows(data?.nodes);
 
-  const columns = useColumns<OutboundRowFragment>(
-    [
-      GenericColumnKey.Selection,
-      [
-        getNameAndColorColumn(),
-        { setter: onUpdate, defaultHideOnMobile: true },
-      ],
-      [
-        'status',
-        {
-          formatter: status =>
-            getStatusTranslator(t)(status as InvoiceNodeStatus),
-        },
-      ],
-      [
-        'invoiceNumber',
-        { description: 'description.invoice-number', width: 150 },
-      ],
-      'createdDatetime',
-      {
-        description: 'description.customer-reference',
-        key: 'theirReference',
-        label: 'label.reference',
-        Cell: TooltipTextCell,
-        width: 175,
-        defaultHideOnMobile: true,
-      },
-      getCommentPopoverColumn(),
-      [
-        'totalAfterTax',
-        {
-          accessor: ({ rowData }) => rowData.pricing.totalAfterTax,
-          width: 125,
-          defaultHideOnMobile: true,
-        },
-      ],
-    ],
-    { onChangeSortBy: updateSortQuery, sortBy },
-    [sortBy]
-  );
+  const { data, isLoading } = useOutbound.document.list(queryParams);
+  const { mutate: onUpdate } = useOutbound.document.update();
 
   const mrtColumns = useMemo(
     (): ColumnDef<OutboundRowFragment>[] => [
       {
         header: t('label.name'),
         accessorKey: 'otherPartyName',
-        ...getNameAndColorSetterColumn<OutboundRowFragment>(
-          onUpdate,
-          isOutboundDisabled
-        ),
+        size: 400,
         enableColumnFilter: true,
         enableSorting: true,
         defaultHideOnMobile: true,
+        Cell: ({ row }) => (
+          <NameAndColorSetterCell
+            onColorChange={onUpdate}
+            getIsDisabled={isOutboundDisabled}
+            row={row.original}
+          />
+        ),
```

Comment:
agreed pattern was just to define cells, and leave the rest of the column definition up to the implementation, as shown here.

this is a bit simpler that the `...getNameAndColorSetterColumn`, easier to understand implications